example-application/api.js

```diff
@@ -1,10 +1,11 @@
 const express = require('express');
 const util = require('util');
-const axios = require('axios');
 const bodyParser = require('body-parser');
 const mailer = require('./libraries/mailer');
 const OrderRepository = require('./data-access/order-repository');
 const errorHandler = require('./error-handling').errorHandler;
+const { getUser } = require('./libraries/users-service');
```

Comment:
Not sure that I would create a lib for this - It's not an organizational Microservice infrastructure rather demo code, a function could be as good

If its npm lib, is it a valid practice to intercept the URL? We decided that not...

Reply:
I'm ok with function, where would you put it?

> If its npm lib, is it a valid practice to intercept the URL? We decided that not...

I put it as lib because I think about it as an SDK for the microservice. (we do that a lot so other teams won't need to re-invent the wheel.
But again, I'm totally fine with a function, just don't know where to put it?